scripts/dev/ci/travis_run.sh

```diff
@@ -14,6 +14,16 @@ elif [[ $TESTENV == eslint ]]; then
     # travis env
     cd qutebrowser/javascript || exit 1
     eslint --color --report-unused-disable-directives .
+elif [[ $TESTENV == shellcheck ]]; then
```

Comment:
This all seems like overcomplicated and like it'd break with filenames with spaces in them - why not simply have newline-separated filenames (i.e. without `-print0` and `xargs` involved) and use `IFS="\n"` instead?

Reply:
You're right, it would filenames containing spaces would probably break this contraption. I will rework it.